app_backend/app/routers/friends.py

```diff
@@ -14,6 +14,36 @@
 router = APIRouter(prefix="/friends")
 security = JWTBearer() 
 
+"""
+    search_friend implements the /search-friend route
+
+    resp: Response
+        The response to send back to the user which contains the status code and body
+
+    friend: AddFriend
+        Contains the email of the user that will be searched 
+
+    returns Response
+        Response.body: dict
+            Provides any msgs/errs for the request
+        Response.status: int
+            The status code for the request
+"""
```

Comment:
This part of the code looks good. Are you currently running tests?

Reply:
Nah I didn't write a test yet mainly because I was under a time crunch.